### PR TITLE
244 add fields create tool page

### PR DESCRIPTION
**Pull request by rishikeshDpp**

### CODE REVIEW

File: src/stylesheets/components/_avatar_image.scss [hunk 1]
```diff
@@ -37,6 +37,30 @@
...
+    width: 160px;
+    height: 100px;
+    flex-shrink: 0;
+    display: grid;
+    place-items: center;
+    position: relative;
+  }
+
+  &--selected {
+    border: 4px solid #8ecb58;
```
Comment (mohitrajane) [inline src/stylesheets/components/_avatar_image.scss:56]: can we use a variable for this color

**Review by mohitrajane — APPROVED** [review]

File: src/stylesheets/components/_avatar_image.scss (hunk 1 above)
Comment (rishikeshDpp) [reply src/stylesheets/components/_avatar_image.scss:56]: I actually pasted this from the css above it in the same file it should be updated there as well, I wanted these styles to be polished by pratheesh
I think he would like to adjust sizing and spacing of the upload buttons and error message spans

Comment (rishikeshDpp) [reply src/stylesheets/components/_avatar_image.scss]: --selected-border-color added

**Comment by mohitrajane** [comment]
closes #224

**Pull request merged**